### ISSUE TITLE
fivetran_webhook triggers update when there are no changes and creates an error

### ISSUE THREAD

**Issue by adrian-montero-hkkd** [fivetran/terraform-provider-fivetran#239]
**Describe the bug**
The first time running a webhook creation it succeeds, but if you run a second apply in tf cloud with no changes to the resource, sometimes it will fail.
When it fails, the second time it shows a change/update in the secret, although it was not changed: 
![image](https://github.com/fivetran/terraform-provider-fivetran/assets/145811752/0d353742-fc50-4725-86b5-9e5470ebe8fd)

If I delete the webhook via fivetran rest api and manually remove the resource from the tf state using tf state rm, and trigger a new run, all the time it works the first time.

My guess is that when the provider queries the webhook to check current state it returns: 
"secret": "******",
and that doesn't match the actual secret string in the terraform file and triggers the update condition.

**To Reproduce**
```
resource "fivetran_webhook" "tf_webhook_prod" {
  type = "group"
  group_id = fivetran_group.fivetran_group.id
  url = "https://myurl.site.com"
  active = true
  events = [
    "sync_start",
    "sync_end",
    "create_connector",
    "pause_connector",
    "resume_connector",
    "delete_connector",
    "connection_successful",
    "connection_failure"
    ]
  secret = "<redacted>"
}
```

**Expected behavior**
If there are no changes in the webhook do not trigger an update.

**Plugin version:**
1.1.5

**Comment by beevital** [comment]
@adrian-montero-hkkd Fix included in latest version v1.1.11
If the fix doesn't work for you feel free to re-open the issue.

**Comment by adrian-montero-hkkd** [comment]
@beevital It seems the issue is reappearing, I had over 200+ webhooks setup and did no change to them and now all the webhooks show as "update" in plan although I did not change anything on them. So in the plan I get all the webhooks with this change: 
![image](https://github.com/fivetran/terraform-provider-fivetran/assets/145811752/5ba35541-f08a-4236-aba3-184b5f06b1db)
And later get this result in the apply:
![image](https://github.com/fivetran/terraform-provider-fivetran/assets/145811752/56cea334-34bb-4016-8d8e-55faee4984e7)
After I apply the plan and re-run a plan-apply cycle with no changes I get yet again this in the plan:
![image](https://github.com/fivetran/terraform-provider-fivetran/assets/145811752/5ba35541-f08a-4236-aba3-184b5f06b1db)
Just like if in the previous cycle that attribute was not changed in the webhook.
